### PR TITLE
Feature query update

### PR DESCRIPTION
**Pull request by vikipretium**
> # [Vet clinic database: query and update animals table](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#vet-clinic-database-query-and-update-animals-table)
> * **In schema.sql:**
>   
>   * I added a column `species` of type `string` to your `animals` table. Modify your `schema.sql` file. ✔️ 
> * **In data.sql:**
>   
>   * I inserted the following data :
>     
>     * Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape. ✔️ 
>     * Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times. ✔️ 
>     * Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times. ✔️ 
>     * Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once. ✔️ 
>     * Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times. ✔️ 
>     * Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times. ✔️ 
>     * Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times. ✔️ 
> * I created transactions to modify the species column and practiced ROLLBACK, SAVEPOINT, and COMMIT. ✔️ 
> * **In queries.sql:**
>   
>   * I wrote queries for the following:
>     
>     * How many animals are there? ✔️ 
>     * How many animals have never tried to escape? ✔️ 
>     * What is the average weight of animals? ✔️ 
>     * Who escapes the most, neutered or not neutered animals? ✔️ 
>     * What is the minimum and maximum weight of each type of animal? ✔️ 
>     * What is the average number of escape attempts per animal type of those born between 1990 and 2000? ✔️ 
> 
> Thanks for your time to review my PR. 👍🏼

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -2,9 +2,75 @@
...
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO backup;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+WHERE weight_kg < 0 ;
+
+COMMIT;
```
Comment (Lordkaito) [inline data.sql]: - [x] Please make sure to move this code into your `queries.sql` file, as this is part of the queries and not the data itself.

File: data.sql [hunk 2]
```diff
@@ -2,9 +2,75 @@
...
+
 INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg)
-VALUES ('Devimon','2017-5-12',5,true,11);
\ No newline at end of file
+VALUES ('Ditto','2022-5-14',4,true,22);
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```
Comment (Lordkaito) [inline data.sql]: - [x] Please make sure to check if your queries are working before every `COMMIT` or `ROLLBACK`. Here, you need to do something like `SELECT species from animals` to make sure it is working as expected.
- [x] Please make sure to apply this to every query so you can see the changes before going back or committing.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @vikipretium, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (vikipretium) [reply data.sql]: Sure @Lordkaito. I'll do it. 👍🏼

File: data.sql (hunk 2 above)
Comment (vikipretium) [reply data.sql]: I'll check all the queries before committing or rollback.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**